packages/rocketchat-livechat/imports/server/rest/visitors.js

```diff
@@ -30,3 +30,31 @@ RocketChat.API.v1.addRoute('livechat/visitor/:visitorToken/room', { authRequired
 		return RocketChat.API.v1.success({ rooms });
 	}
 });
+
+RocketChat.API.v1.addRoute('livechat/visitor/:visitorToken/room/transfer', { authRequired: true }, {
+	post() {
+		if (!RocketChat.authz.hasPermission(this.userId, 'view-livechat-manager')) {
+			return RocketChat.API.v1.unauthorized();
+		}
+
+		const rooms = RocketChat.models.Rooms.findOpenByVisitorToken(this.urlParams.visitorToken).fetch();
```

Comment:
I'm not sure about this approach.. If you're transfer a Livechat Room to other agent, you need to know which room you want to transfer, don't you think? Otherwise, it looks like a random transfer.
I think that passing the `roomId` by param would be better.